_commands/basics/sleep.md

```diff
@@ -3,22 +3,37 @@
 
 sleep
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+The sleep command makes your script, terminal, or anything else that is bash executable go to sleep for however long you specify 
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+sleep 5s
 ~~~
-
+ 
 <!--more-->
 
 ### Useful Options / Examples
+The default command is sleep __NUMBER__ __OPTION__ and the number is interpreted by bash as in seconds however in the option arguement you can specify the sleep delay in seconds, minutes, hours, or days 
+
+s - seconds
```

Comment:
could you enclose each flag with backticks so they render like `s` instead of s? Also you'll need to enter an extra new line between each list element, like so:
```
`s` - ...

`m` - ...

`h` - ...

`d` - ...
```